src/asm/rgbasm.5

```diff
@@ -0,0 +1,426 @@
+value.
+If you are familiar with C you can think of it as the same as #define.
+.Pp
+.Dl COUNTREG EQUS "[hl+]"
+.Dl ld a,COUNTREG
+.Pp
+.Dl PLAYER_NAME EQUS \*q\[rs]\*qJohn\[rs]\*q\*q
+.Dl db PLAYER_NAME
+.Pp
+Note that : following the label-name is not allowed, and that strings must be
+quoted to be useful.
+.Pp
+This will be interpreted as:
+.Pp
+.Dl ld a,[hl+]
+.Dl db \*qJohn\*q
+.Pp
+String-symbols can also be used to define small one-line macros:
+.Pp
+.Dl PUSHA EQUS \*qpush af\[rs]npush bc\[rs]npush de\[rs]npush hl\[rs]n\*q
+.Pp
+Note that a colon (:) following the label-name is not allowed.
+String equates can't be exported or imported.
+.Pp
+.It Sy MACRO
+.Pp
+###TODO###
+.Pp
+.El
+.Ss Exporting and importing symbols
+###TODO###
+
+
+
+
+.Sh MISCELLANEOUS
+.Ss Changing options while assembling
+.Ar OPT No can be used to change some of the options during assembling the
+source instead of defining them on the commandline.
+.Pp
+.Ar OPT No takes a comma-seperated list of options as its argument:
+.Pp
+.Dl PUSHO
+.Dl OPT   g.oOX    ;Set the GB graphics constants to use these characters
+.Dl DW    `..ooOOXX
+.Dl POPO
+.Dl DW    `00112233
+.Pp
+The options that OPT can modify are currently:
+.Sy b , e No and Sy g .
+.Pp
+.Ar POPO No and Ar PUSHO No provide the interface to the option stack.
+.Ar PUSHO No will push the current set of options on the option stack.
+.Ar POPO No can then later be used to restore them.
+Useful if you want to change some options in an include file and you don't want
+to destroy the options set by the program that included your file.
+The stacks number of entries is limited only by the amount of memory in your
+machine.
+.Sh ALPHABETICAL LIST OF KEYWORDS
+.Bl -inset -compact
+.It Sx @
+.It Sx __DATE__
+.It Sx __FILE__
+.It Sx __LINE__
+.It Sx __TIME__
+.It Sx _NARG
+.It Sx _PI
+.It Sx _RS
+.It Sx ACOS
+.It Sx ASIN
+.It Sx ATAN
+.It Sx ATAN2
+.It Sx BANK
+.It Sx COS
+.It Sx DB
+.It Sx DEF
+.It Sx DIV
+.It Sx DS
+.It Sx DW
+.It Sx ELSE
+.It Sx ENDC
+.It Sx ENDM
+.It Sx ENDR
+.It Sx EQU
+.It Sx EQUS
+.It Sx EXPORT
+.It Sx FAIL
+.It Sx GLOBAL
+.It Sx HIGH
+.It Sx HRAM
+.It Sx IF
+.It Sx INCBIN
+.It Sx INCLUDE
+.It Sx LOW
+.It Sx MACRO
+.It Sx MUL
+.It Sx OPT
+.It Sx POPO
+.It Sx POPS
+.It Sx PRINTF
+.It Sx PRINTT
+.It Sx PRINTV
+.It Sx PURGE
+.It Sx PUSHO
+.It Sx PUSHS
+.It Sx REPT
+.It Sx RB
+.It Sx ROM0
+.It Sx ROMX
+.It Sx RSRESET
+.It Sx RSSET
+.It Sx RW
+.It Sx SECTION
+.It Sx SET
+.It Sx SHIFT
+.It Sx SIN
+.It Sx SRAM
+.It Sx STRCAT
+.It Sx STRCMP
+.It Sx STRIN
+.It Sx STRLEN
+.It Sx STRLWR
+.It Sx STRSUB
+.It Sx STRUPR
+.It Sx TAN
+.It Sx VRAM
+.It Sx WRAM0
+.It Sx WRAMX
+.It Sx WARN
+.El
+.Sh SEE ALSO
+.Xr rgbasm 1 ,
+.Xr rgblink 1 ,
+.Xr rgblink 5 ,
+.Xr rgbds 7 ,
+.Xr gbz80 7
+.Sh HISTORY
+.Nm rgbds
+was originally written by Carsten S\(/orensen as part of the ASMotor package,
+and was later packaged in RGBDS by Justin Lloyd.
+It is now maintained by a number of contributors at
+https://github.com/rednex/rgbds.
```

Comment:
I've done this in https://github.com/rednex/rgbds/commit/466bb9ed0b1497e33dfcfa458ceafd11172ac9a0 It will be in the next release (fix or no fix in troff).

It's true that it looks wrong in the terminal, but the HTML documentation looks better.